library/src/main/java/com/pokegoapi/api/map/pokemon/CatchResult.java

```diff
@@ -93,4 +93,9 @@ public boolean isFailed() {
 		}
 		return (this.getStatus() != CatchStatus.CATCH_SUCCESS || failed);
 	}
+
+	public boolean hasCaptureAward()
+	{
```

Comment:
Yup that's what I'm talking about. Java standard is to not wrap the bracket to next line after method signature. It's only style issue ;)